clusterloader2/testing/load/config.yaml

```diff
@@ -17,13 +17,13 @@
 {{$ENABLE_CHAOSMONKEY := DefaultParam .ENABLE_CHAOSMONKEY false}}
 {{$PROMETHEUS_SCRAPE_KUBE_PROXY := DefaultParam .PROMETHEUS_SCRAPE_KUBE_PROXY true}}
 {{$ENABLE_PROMETHEUS_API_RESPONSIVENESS := DefaultParam .ENABLE_PROMETHEUS_API_RESPONSIVENESS false}}
-{{$ENABLE_CONFIGMAPS := DefaultParam .ENABLE_CONFIGMAPS false}}
-{{$ENABLE_DAEMONSETS := DefaultParam .ENABLE_DAEMONSETS false}}
-{{$ENABLE_JOBS := DefaultParam .ENABLE_JOBS false}}
-{{$ENABLE_PVS := DefaultParam .ENABLE_PVS false}}
-{{$ENABLE_SECRETS := DefaultParam .ENABLE_SECRETS false}}
-{{$ENABLE_STATEFULSETS := DefaultParam .ENABLE_STATEFULSETS false}}
-{{$ENABLE_NETWORKPOLICIES := DefaultParam .ENABLE_NETWORKPOLICIES false}}
```

Comment:
Please don't touch networkpolicies, I think they are not ready yet.
Sorry, it should've been mentioned in the issue

Reply:
reverted network policy changes. Default set to false again